services/src/filesystems/virtual_file_system.adb

```diff
@@ -61,9 +63,40 @@ package body Virtual_File_System is
                     Path       : Pathname)
                     return Status_Kind
    is
-      pragma Unreferenced (This, Path);
+      procedure Destroy is new Ada.Unchecked_Deallocation
```

Comment:
Ravenscar-full has __gnat_free, but no Ravenscar-sfp. As I said we should provide both heap and pool allocation policies.